AsyncDisplayKit/Details/ASImageProtocols.h

```diff
@@ -70,10 +70,10 @@ typedef NS_ENUM(NSUInteger, ASImageDownloaderPriority) {
  @param URL The URL of the image to download.
  @param callbackQueue The queue to call `downloadProgressBlock` and `completion` on.
  @param downloadProgress The block to be invoked when the download of `URL` progresses.
- @param progress The progress of the download, in the range of (0.0, 1.0), inclusive.
+   @param progress The progress of the download, in the range of (0.0, 1.0), inclusive.
```

Comment:
The parameters are indented to differentiate the methods parameters from the parameters of parameters. Ex `progress` is a parameter of the `downloadProgress` parameter, not the method itself. I will flatten the indentation if that's prefered.